requirements/testing/travis_all.txt

```diff
@@ -1,15 +1,10 @@
 # pip requirements for all the travis builds
 
 coverage
-cycler
-numpy
```

Comment:
Why don't we need these anymore?

Reply:
They are matplotlib dependencies and will be installed anyway, the same change was proposed by @anntzer  https://github.com/matplotlib/matplotlib/pull/16117#discussion_r366865308

Reply:
Correct, we should only specify these if we want to actively pin the back to an old version.